asn1/CommonLedgerProtocol.asn

```diff
@@ -0,0 +1,107 @@
+CommonLedgerProtocol
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+IMPORTS
+    UInt8,
+    UInt32,
+    UInt64,
+    UInt128,
+    UInt256
+    FROM GenericTypes
+
+    Address,
+    Timestamp
+    FROM InterledgerTypes
+
+    InterledgerPacket
+    FROM GenericPacket
+;
+
+SideProtocolData ::= SEQUENCE OF SEQUENCE {
+  protocolName IA5String,
+  protocolData OCTET STRING
+}
+
+-- Ack and Response are the response types.
+-- When using these in a CLP packet, the requestId should match
+-- the requestId of the request they respond to.
+
+Ack ::= SideProtocolData
```

Comment:
It's also how clp-packet [implements it currently](https://github.com/interledgerjs/clp-packet/blob/master/index.js#L75-L80)